package.json

```diff
@@ -48,10 +53,11 @@
     "@types/react-lazylog": "^4.5.0",
     "@types/react-virtualized": "^9.21.9",
     "@types/uuid": "^8.3.0",
-    "chalk": "^2.4.2",
+    "chalk": "^4.0.0",
     "escodegen": "^1.11.1",
-    "espree": "^3.5.4",
-    "estraverse": "^4.2.0",
+    "espree": "^7.0.0",
+    "estraverse": "^5.1.0",
+    "fbemitter": "^2.1.1",
```

Comment:
i did some work recently to remove fbemitter (it has a dependency that has some minor vulnerability)

is there a reason we need this back?

Reply:
Nope, this was my mistake when resolving the merge conflict.